packages/configuration-validation/CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+# 1.1.0
+
+### Features
+
+- [#954](https://github.com/okta/okta-oidc-js/pull/954) Add ES entry point for configuration-validation
+
+# 1.0.0
+
+### Features
+
+- [#879](https://github.com/okta/okta-oidc-js/pull/879) Convert to Typescript
```

Comment:
prefer `Adds support for Typescript` as we want to tell the users that we support types now instead of mentioning the internal code change.